A-4E-C/Weapons/A4E_Weapons.lua

```diff
@@ -1964,9 +1964,9 @@ end
 local dispenser_data =
 {
     --use shapename,         bomblet,          bomblet_count
-    ["CBU-1/A"]          = { bomblet = BLU_4B_NEW_GROUP, bomblet_count = 19 },
-    ["CBU-2/A"]          = { bomblet = BLU_3B_NEW_GROUP, bomblet_count = 19 },
-    ["CBU-2B/A"]         = { bomblet = BLU_3B_NEW_GROUP, bomblet_count = 19 },
+    ["CBU-1/A"]          = { bomblet = BLU_4B_NEW_GROUP, bomblet_count = 19, tube_size=tube_size_1A },
+    ["CBU-2/A"]          = { bomblet = BLU_3B_NEW_GROUP, bomblet_count = 19, tube_size=tube_size_2A },
```

Comment:
I think you are right here but have you checked in game?

Reply:
Yep, here are values pre-change (in lb):
| CBU 1/A      | CBU 2/A      | CBU 2B/A      |
| ------------ | ------------ | ------------- |
| 743          | 12763        | 12763         |
| CBU 1/A \* 2 | CBU 2/A \* 2 | CBU 2B/A \* 2 |
| 406          | 1691         | 1691          |

and post-change:
| CBU 1/A      | CBU 2/A      | CBU 2B/A      |
| ------------ | ------------ | ------------- |
| 743          | 758          | 758           |
| CBU 1/A \* 2 | CBU 2/A \* 2 | CBU 2B/A \* 2 |
| 1592         | 1623         | 1823          |

Those values are *close enough* to the cbu datasheet.
There is still a change to do for CBU 2/A and CBU 2B/A versions (both use same tube_size, hence same weight, as shown in the code below), I planned it for later to reduce the number of changes for this PR.

```
local bomblet_data =
{
    --use shapename,        mass,               wstype,                                                drag,
    ["BLU-3B"]          = { mass = BLU_3B_NEW_GROUP.mass * tube_size_2A, wstype = BLU_3B_OLD.wsTypeOfWeapon, model = BLU_3B_NEW_GROUP.model, cx = 0.0001, len = 0.095+0.05, clsid = BLU_3B_NEW_GROUP.clsid },
    ["BLU-4B"]          = { mass = BLU_4B_NEW_GROUP.mass * tube_size_1A, wstype = BLU_4B_OLD.wsTypeOfWeapon, model = BLU_4B_NEW_GROUP.model, cx = 0.0001, len = 0.095+0.05, clsid = BLU_4B_NEW_GROUP.clsid },
}```

Reply:
Beautiful great work.